src/core/util/env.js

```diff
@@ -87,13 +87,17 @@ export const nextTick = (function () {
   }
 
   return function queueNextTick (cb: Function, ctx?: Object) {
-    const func = ctx
-      ? function () { cb.call(ctx) }
-      : cb
-    callbacks.push(func)
-    if (!pending) {
-      pending = true
-      timerFunc()
+    if (cb) {
+      var func = ctx
+        ? function () { cb.call(ctx) }
```

Comment:
Well, if you changed cb to be nullable, then flow will report error here because it is pessimistic